cli/sawtooth_cli/main.py

```diff
@@ -133,6 +137,9 @@ def main_wrapper():
     # pylint: disable=bare-except
     try:
         main()
+    except AssertionError as e:
```

Comment:
@vaporos I believe I resolved this. Main wrapper now does not catch the AssertionError.